chef/cookbooks/provisioner/libraries/repositories.rb

```diff
@@ -22,16 +22,24 @@ class << self
       # they can be used.
       def get_repos(platform, version, arch)
         repositories = {}
+        db = "repos-#{platform}-#{version}-#{arch}".gsub(".", "_")
```

Comment:
Performance/StringReplacement: Use `tr` instead of `gsub`. (https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)